src/v/storage/probe.cc

```diff
@@ -281,7 +282,9 @@ void probe::delete_segment(const segment& s) {
 }
 
 void probe::batch_write_error(const std::exception_ptr& e) {
-    stlog.error("Error writing record batch {}", e);
+    auto level = ssx::is_shutdown_exception(e) ? ss::log_level::warn
```

Comment:
This is already logged once at https://github.com/redpanda-data/redpanda/blob/dd70a59a76f8b40e97a2bfe65c1a97270faaa081/src/v/storage/disk_log_appender.cc#L119 (caller).

Shall we remove the log line here (in this diff) completely and change the log level on the other one?